scss/blocks/tabs.scss

```diff
@@ -228,13 +229,17 @@
   background-color: transparent;
   color: var(--colorInkPrimary);
   border-top: 1px solid var(--colorBorderColorSubtle);
-  border-bottom: 1px solid var(--colorBorderColorSubtle);
+  border-bottom: 1px solid transparent;
```

Comment:
let's leave this as it was there originally unless we have a reason to change :)